app/services/cookies_consent.rb

```diff
@@ -0,0 +1,28 @@
+class CookiesConsent
```

Comment:
IMO this should also be inside the engine and under its namespace. Then, it should be available  elsewhere as `Web::CookiesConsent`

Reply:
agreed.